x/json_test.go

```diff
@@ -0,0 +1,116 @@
+package x
+
+import (
+	"testing"
+
+	"github.com/mohae/deepcopy"
+	"github.com/stretchr/testify/require"
+)
+
+type TestType struct {
+	Field1 string
+	Field2 []string
+	Field3 struct {
+		Field1 bool
+		Field2 []int
+	}
+}
+
+func TestApplyJSONPatch(t *testing.T) {
+	object := TestType{
+		Field1: "foo",
+		Field2: []string{
+			"foo",
+			"bar",
+			"baz",
+			"kaz",
+		},
```

Comment:
The same for this file, do we change anything specific here from `ory/x`?
https://github.com/ory/x/blob/master/jsonx/patch_test.go

Reply:
See https://github.com/ory/kratos/pull/2471#discussion_r917421577